client/components/EventMetadata/index.jsx

```diff
@@ -1,82 +1,93 @@
+            <Column>
+                <i className="icon-calendar-list" />
+            </Column>
+            <Column grow={true} border={false}>
+                <Row>
+                    <StateLabel item={event} verbose={true}/>
+                    <span className="sd-overflow-ellipsis sd-list-item--element-grow">
+                        <span className="sd-list-item__text-strong">{event.name}</span>
+                        <time title>{dateStr}</time>
+                    </span>
 
-            if (value) {
-                return k[1](value);
-            }
-        }
-    }
-    const keysContainValue = some(keys, (k) => getValue(k));
+                </Row>
+            </Column>
+        </Item>
+    );
+
+    const eventInDetailTopBar = (
+        <Item noBg={true} noHover={true}>
+            <Column border={false}>
+                <span className="double-size-icn double-size-icn--light">
+                    <i className="icon-calendar" /></span>
+            </Column>
+            <Column border={false} grow={true}>
+                <Row>
+                    <span className="sd-overflow-ellipsis sd-list-item--element-grow">
+                        <span className="sd-list-item__text-strong">{event.name}</span>
+                    </span>
+                </Row>
+                <Row>
+                    <span className="sd-overflow-ellipsis sd-list-item--element-grow">
+                        <span className="sd-list-item__location">{get(event, 'location[0].name', 'Unknown')}</span>
+                        <time title>{dateStr}</time>
+                    </span>
+                </Row>
+            </Column>
+        </Item>
+    );
 
-    if (keysContainValue) {
-        return [
-            <dt key="dt">{label}</dt>,
-            ...keys.map((key) => (
-                (key === 'definition_long') ? (
-                    <InputTextAreaField key={key}
-                        input={{
-                            value: getValue(key),
-                            name: key,
-                        }}
-                        meta={{}}
-                    />
-                ) : (
-                    <dd key={key}>{getValue(key)}</dd>
-                )
-            )),
-        ];
-    }
-}
-// eslint-disable-next-line react/no-multi-comp
-export function EventMetadata({event}) {
-    return (
+    const eventInDetail = (
         <div>
-            <StateLabel item={event} verbose={true}/>
-            <div className="metadata-view EditPlanningPanel__body--event">
-                <dl>
-                    {FIELDS.map((arrayProps) => renderDict.bind(null, event)(...arrayProps))}
-                </dl>
-            </div>
+            <PreviewRow>
+                <StateLabel item={event} verbose={true}/>
+            </PreviewRow>
+            <PreviewRow label={gettext('Name')} value={event.name} />
+            <PreviewRow>
+                <EventScheduleSummary schedule={event.dates}
```

Comment:
`EventScheduleSummary` already uses `PreviewRow` which having two adds more padding.

`EventScheduleSummary` also has an unnecessary `<div>`, `PreviewRow` can be the parent node for `EventScheduleSummary` and remove `PreviewRow` from `EventMetadata`